### ISSUE TITLE
Docs: Clarify whether durable purge-history can affect functions not being run locally

### ISSUE THREAD

**Issue by spiffytech** [Azure/azure-functions-core-tools#2269]
I need to do cleanup for my functions during local development, and it's unclear from the docs and my attempt to follow the source code whether running `func durable purge-history` can ever affect functions running in Azure. 

I'd appreciate if these could be documented: 

- Any risks of `purge-history` affecting non-local Durable Function instances.
- How to identify all Functions that would be affected by `purge-history`, so I can sanity-check before doing it.

**Comment by anthonychu** [comment]
I believe it looks in host.json and local.settings.json for the task hub name and storage connection string to perform the action. As long as you're not sharing those with a production app, you should be okay. /cc @ConnorMcMahon

**Comment by ConnorMcMahon** [comment]
@anthonychu, that is correct. Any actions taken against any Durable Functions app should only affect a given TaskHub + storage account combination. As long as your local app doesn't share that combination (and our general recommendation for Durable Functions is to use a separate TaskHub + storage account for every running funciton app, including local dev), purge history will only clear your local app's history.

I'm not sure where the best place to document this. @anthonychu any ideas?
